### PR TITLE
Adding bootstrap__etc_hosts to make it possible to disable hostname addition to /etc/hosts

### PR DESCRIPTION
**Pull request by tonin**
Closes debops/ansible-bootstrap#52

### CODE REVIEW

File: ansible/roles/debops.bootstrap/defaults/main.yml [hunk 1]
```diff
@@ -35,6 +35,13 @@ bootstrap__domain: '{{ inventory_hostname.split(".")[1:] | join(".") }}'
 # Set custom DNS hostname on a given host.
 bootstrap__hostname: '{{ inventory_hostname_short | d(inventory_hostname.split(".")[0]) }}'
 
+                                                                   # ]]]
+# .. envvar:: bootstrap__etc_hosts [[[
+#
+# Add the hostname to /etc/hosts and make it point to 127.0.1.1
+# This is usually what you want except if you manage yourself the
+# hostname entry and want it to point to some other IP.
+bootstrap__etc_hosts: True
```
Comment (drybjed) [inline ansible/roles/debops.bootstrap/defaults/main.yml]: Can you add an empty line after the variable? This will make the variable definitions uniform.

Comment (tonin) [reply ansible/roles/debops.bootstrap/defaults/main.yml]: Added.

**Comment by drybjed** [comment]
Closing and reopening to see if this triggers Travis-CI.

**Comment by drybjed** [comment]
I'm not sure why Travis-CI doesn't want to pick this PR up... Changes are small enough that I think it will be OK if I merge this, code should be tested on the `master` branch as well by Travis.

**Pull request merged**